### PR TITLE
Add changes summary for 0.8

### PR DESCRIPTION
**Pull request by hoechenberger**

### CODE REVIEW

File: doc/whats_new.rst [hunk 1]
```diff
@@ -13,6 +13,32 @@ What's new?
 Version 0.8 (2021-07-14)
 ------------------------
 
+This release brings numerous improvements and fixes based on feedback from our
+users, including those working with very large datasets. MNE-BIDS now handles
+previously-overlooked edge cases, offers a much more efficient way to
+store data on macOS and Linux (using symbolic links), and lays the groundwork
+for supporting derivatives, i.e., storing modified data.
```
Comment (hoechenberger) [inline doc/whats_new.rst:20]:
```suggestion
for supporting BIDS derivatives, i.e., storing modified data.
```

File: doc/whats_new.rst [hunk 2]
```diff
@@ -13,6 +13,32 @@ What's new?
...
+  entities. What this means is that you can now do things like
+  ``bids_path.subject = '01'`` instead of ``bids_path.update(subject='01')``.
+- We now support Deep Brain Stimulation (DBS) data.
+- The way we handle anatomical landmarks was greatly revamped to ensure we're
+  always using the correct coordinate systems. A new function,
+  `mne_bids.get_anat_landmarks`, helps with extracting fiducial points from
+  anatomical scans.
+- When creating a BIDS dataset from FIFF files on macOS and Linux, MNE-BIDS
+  can now optionally generate symbolic links to the original files instead of
+  creating copies. Simply pass ``symlink=True`` to
```
Comment (hoechenberger) [inline doc/whats_new.rst:38]:
```suggestion
  copies. Simply pass ``symlink=True`` to
```

**Review by sappelhoff — COMMENTED** [review]
thanks, that's great! I'll finish up now

File: doc/whats_new.rst [hunk 3]
```diff
@@ -13,6 +13,32 @@ What's new?
 Version 0.8 (2021-07-14)
```
Comment (hoechenberger) [inline doc/whats_new.rst:13]:
```suggestion
Version 0.8 (2021-07-15)
```

**Review by sappelhoff — APPROVED** [review]
merge when you are done @hoechenberger

**Pull request merged**